app/Formatter/BoardSwimlaneFormatter.php

```diff
@@ -101,6 +101,7 @@ public function format()
             foreach ($swimlane['columns'] as $columnIndex => &$column) {
                 $column['column_nb_tasks'] = $this->swimlanes[0]['columns'][$columnIndex]['column_nb_tasks'];
                 $column['column_nb_score'] = $this->swimlanes[0]['columns'][$columnIndex]['column_score'];
+                $column['column_nb_open_tasks'] = $this->taskFinderModel->countByColumnId($column['project_id'], $column['id'] );
```

Comment:
That might generate too many SQL queries for large board. Need to find another way of fetching the number of open tasks.

Reply:
@fguillot I've changed the implementation, it should be a single select now.